packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -449,7 +452,8 @@ export async function runNpmInstall(
     debug(`Installing to ${destPath}`);
 
     const opts: SpawnOptionsExtended = { cwd: destPath, ...spawnOpts };
-    const env = opts.env ? { ...opts.env } : { ...process.env };
+    const envObj = opts.env || process.env;
+    const env: typeof process.env = { PATH: envObj.PATH, ...envObj };
```

Comment:
I don't think `envObj.PATH` will work. It needs to be `process.env.PATH` because we had to fix this here:

- https://github.com/vercel/vercel/pull/7881

Reply:
Seems to work in both the unit tests and the Node.js REPL:

```
C:\Users\chris\projects\vercel>node
Welcome to Node.js v16.16.0.
Type ".help" for more information.
> const envObj = process.env;
undefined
> envObj.PATH
C:\Program Files (x86)\Razer Chroma SDK\bin;C:\Program Files\Razer Chroma SDK\bin;C:\Program Files (x86)\Razer\ChromaBroadcast\bin;C:\Program Files\Razer\ChromaBroadcast\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\WindowsApps;C:\Users\chris\AppData\Roaming\nvm;C:\Program Files\nodejs;C:\Program Files\Go\bin;C:\Program Files\Git\cmd;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Users\chris\.cargo\bin;C:\Users\chris\AppData\Local\Microsoft\WindowsApps;C:\Users\chris\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\chris\AppData\Roaming\nvm;C:\Program Files\nodejs;C:\Users\chris\go\bin
> process.env.PATH === envObj.PATH
true
> envObj.Path === envObj.PATH
true
```

Reply:
Oh yeah I see now the issue in the previous PR was that opts was already a clone haha